platform-server-kernel/src/main/resources/db/migration/20160321_120952__language.sql

```diff
@@ -0,0 +1,28 @@
+--
+-- Table structure for table `language`
+--
+
+DROP TABLE IF EXISTS `language`;
+CREATE TABLE `language` (
+  `id` bigint(20) NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) DEFAULT NULL,
+  `tag` datetime DEFAULT NULL,
```

Comment:
`Caused by: org.hibernate.tool.schema.spi.SchemaManagementException: Schema-validation: wrong column type encountered in column [tag] in table [language]; found [datetime (Types#TIMESTAMP)], but expecting [varchar(255) (Types#VARCHAR)]`

your tag is a datetime :D